netbout-web/src/test/resources/META-INF/MANIFEST.MF

```diff
@@ -23,3 +23,6 @@ Netbout-SmtpPassword: test
 Netbout-Basic-Realm: testrealm
 Netbout-Basic-User: testuser
 Netbout-Basic-Pwd: testpassword
+Netbout-Invite-Content: "You are invited into the Netbout click on the link to register"
+Netbout-Invite-Url: "http://www.netbout.com/b/%d?invite=%s"
```

Comment:
@mstzn36 good idea, let' rename it and build the url in the class
`Netbout-Site: "www.netbout.com"`